MonoZelda/Scenes/DungeonScene.cs

```diff
@@ -61,30 +61,25 @@ public void LoadContent(ContentManager contentManager)
         // create spritedict to pass into player controller
         var playerSpriteDict = new SpriteDict(contentManager.Load<Texture2D>(TextureData.Player), SpriteCSVData.Player, 1, new Point(100, 100));
         player.SetPlayerSpriteDict(playerSpriteDict);
-
-        //create some sample hitboxes
-        Collidable itemHitbox1 = new Collidable(new Rectangle(100, 200, 50, 50), graphicsDevice, CollidableType.Item);
-        collisionController.AddCollidable(itemHitbox1);
-        Collidable itemHitbox2 = new Collidable(new Rectangle(200, 200, 100, 100), graphicsDevice, CollidableType.Item);
-        collisionController.AddCollidable(itemHitbox2);
-        Collidable itemHitbox3 = new Collidable(new Rectangle(350, 250, 50, 50), graphicsDevice, CollidableType.Item);
-        collisionController.AddCollidable(itemHitbox3);
     }
 
 
+
     private void LoadRoom(ContentManager contentManager)
     {
         LoadRoomTextures(contentManager);
         CreateStaticColliders();
-        SpawnItems();
+        SpawnItems(contentManager);
```

Comment:
This must be what spawns the items when a new room loads.

Reply:
Correct